--- claws-mail-3.5.0/src/addressbook.c	2008-06-27 09:40:25.000000000 +0200
+++ claws-mail-3.5.0/src/addressbook.c-new	2008-09-20 19:27:43.778679758 +0200
@@ -35,7 +35,7 @@
 #include <gtk/gtklabel.h>
 #include <gtk/gtkentry.h>
 #include <gtk/gtkctree.h>
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 #include <gtk/gtktable.h>
 #include <gtk/gtkhbbox.h>
 #include <gtk/gtkbutton.h>
--- claws-mail-3.5.0/src/folderview.c	2008-06-27 09:40:20.000000000 +0200
+++ claws-mail-3.5.0/src/folderview.c-new	2008-09-20 19:28:14.111593792 +0200
@@ -26,7 +26,7 @@
 #include <gtk/gtkscrolledwindow.h>
 #include <gtk/gtkctree.h>
 #include <gtk/gtkcontainer.h>
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 #include <gtk/gtkstyle.h>
 #include <gtk/gtksignal.h>
 #include <gtk/gtkmain.h>
--- claws-mail-3.5.0/src/gtk/gtksctree.h	2008-06-27 09:40:18.000000000 +0200
+++ claws-mail-3.5.0/src/gtk/gtksctree.h-new	2008-09-20 19:25:56.922963495 +0200
@@ -4,7 +4,6 @@
 #define __GTK_SCTREE_H__
 
 #include <gtk/gtk.h>
-#include <gtk/gtkclist.h>
 #include <gtk/gtkctree.h>
 
 /* This code is based on "gtkflist.{h,c}" from mc-4.5.42 .*/
--- claws-mail-3.5.0/src/plugins/pgpcore/select-keys.c	2008-06-27 09:40:25.000000000 +0200
+++ claws-mail-3.5.0/src/plugins/pgpcore/select-keys.c-new	2008-09-20 19:40:53.759947514 +0200
@@ -33,7 +33,7 @@
 #include <gtk/gtkscrolledwindow.h>
 #include <gtk/gtkvbox.h>
 #include <gtk/gtkhbox.h>
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 #include <gtk/gtklabel.h>
 #include <gtk/gtkentry.h>
 #include <gtk/gtkhbbox.h>
