--- sylpheed-claws-0.9.12/configure.ac.orig	Mon Jun 28 18:04:41 2004
+++ sylpheed-claws-0.9.12/configure.ac	Tue Jul 20 16:33:06 2004
@@ -90,7 +90,7 @@
 AM_ICONV
 
 dnl for gettext
-ALL_LINGUAS="bg cs de el en_GB es fr hr hu it ja ko nl pl pt_BR ru sk sr sv zh_CN zh_TW.Big5"
+ALL_LINGUAS="bg cs de el en_GB es fr hr hu it ja ko nl pl pt_BR ru sk sr sv zh_CN zh_TW"
 AM_GNU_GETTEXT([use-libtool])
 AM_GNU_GETTEXT_VERSION(0.12.1)
 dnl AC_CHECK_FUNC(gettext, AC_CHECK_LIB(intl, gettext))
